test/specs/reference/tests/16.default_value.js

```diff
@@ -0,0 +1,94 @@
+
```

Comment:
These are the changes for your other PR that was merged. Why is this part of this PR too?